examples/provider/provider.tf

```diff
@@ -40,6 +40,13 @@ provider "netapp-ontap" {
       password = var.password
       validate_certs = var.validate_certs
     },
+    {
```

Comment:
Please follow other cluster setup replacing the hostname IP with ****2

Reply:
I've made changes to the connection profile.
How can I encrypt the hostname IPs? Is it just replacing it with asterisks leaving the last octet?